autorest/codegen/models/base_model.py

```diff
@@ -22,5 +22,5 @@ def __init__(
     def id(self) -> int:
         return id(self.yaml_data)
 
-    def __repr__(self) ->str:
+    def __repr__(self):
```

Comment:
I'm surprised you remove annotations :)

Reply:
I can add it back in, I just assumed since __repr__ is a defined function that returns the string version of a model it would be redundant to annotate it.